### ISSUE TITLE
Where to place phenotype?

### ISSUE THREAD

**Issue by matentzn** [OBOFoundry/COB#123]
As discussed [here](https://github.com/OBOFoundry/COB/issues/42#issuecomment-684181008), lets start the certainly painful discussion on where in the COB hierarchy phenotype (the biological condition, not to be confused in the information item "phenotypic finding", which already exist).

I would also like to be able to keep this discussion a bit separate from normal/abnormal (thats for the[ Phenotype Reconciliation effort ](https://github.com/obophenotype/upheno/wiki/Phenotype-Ontologies-Reconciliation-Effort)to figure out, not COB IMO). Maybe lets start with gathering some basic "a phenotype is" discussions.

Maybe before we start anything, lets start with a reasonable rough definition on [nature.com](https://www.nature.com/scitable/definition/phenotype-phenotypes-35/) and see how we go from there.

```
The term "phenotype" refers to the observable physical properties of an organism; these include the organism's appearance, development, and behavior. An organism's phenotype is determined by its genotype, which is the set of genes the organism carries, as well as by environmental influences upon these genes.
```

This already shows that defining phenotype as any one thing is going to be complex. Here are some example phenotypes:

- "abnormally increased head size" 
- "erratic behaviour"
- "abnormality of the cardivascular system"
- "behaviour phenotype" (unqualified phenotype -> this is a bit confusing, but often used as grouping classes in FYPO, MP and others)
- "normal behaviour" (indistinguishable from wildtype - please _lets not discuss THIS_ here, that's going to break all cans of worms loose.. We dont need to care about this for COB))

These are some examples of non-phenotypes:

- "head size" (this is a [trait](https://github.com/obophenotype/upheno/issues/716))
- "behaviour" (this is a, I dont know, process or something similar)
- "erratic" (this is a phenotypic quality)

So, this issue is NOT about normal/abnormal/pathological etc. Its about where in the COB hierarchy a `observable physical properties of an organism` should be placed.

@sbello @dosumis @seger @srobb1 @chris-grove @mah11 @nicolevasilevsky @Clare72 @ybradford (please tag all those pheno-heroes I have missed now).

**Comment by sbello** [comment]
The MP defines phenotype as "the observable morphological, physiological, behavioral and other characteristics of mammalian organisms that are manifested through development and lifespan"

The VT defines traits as "any measurable or observable characteristic..."

The key difference being that the phenotype is a realized or manifest trait. So large hands (phenotype) is a manifestation of hand size (trait).
 
That said I think using the COB 'characteristic' parent makes sense for both. 

Are we trying to determine if we can define a parent in COB that distinguishes phenotype from trait? Or do we just want to determine where in the COB tree phenotypes and traits belong?

**Comment by cmungall** [comment]
great start @matentzn - if this were all there was to it then we could declare a phenotype subclass of characteristic (or aggregate of charactertistics, as currently defined, which may itself be a characteristic, needs to be axiomatized) and declare victory.

But as you know we have phenotype ontologies with *labels* that suggest material entities or processes (neoplasm, various deformities, heart attack). Also when questioned, the apparent *intent* of the ontology developers is that these represent material entities or processes or the like. However, the actual *logical definitions, axiomatization, and usage* in external axioms is as aggregates of characteristics. So how do we resolve this problem. One is to stick with the status quo and live with the divergence in *apparent* semantics and *actual* semantics, but this is unsatisfying. Another is to adopt something like the Schulz-union model. Or we just hammer this out and get everyone on the same page... this isn't really a COB thing though

**Comment by dosumis** [comment]
Before the introduction of the has_part buffer, the standard classification would have been in the BFO realizable branch.  COB:characteristic is basically designed to occupy the right place for that now.

**Comment by dosumis** [comment]
> However, the actual logical definitions, axiomatization, and usage in external axioms is as aggregates of characteristics

👍 1

**Comment by cmungall** [comment]
@sbello with the current definitions and pato hierarchy, `blue and characteristic-of some eye` is a subclass of `color and characteristic-of some eye`. The distinction between the two is not something that is captured by a parent class, this is more of a metaclass thing. pato does this with logically silent subset tags.

This all goes back to the modeling in bfo and the determinable/determinant division which bfo chooses to collapse. I was in favor at the time but it's not always intuitive. We could revisit this - and restore pato back to two hierarchies.

alternatively, there are other ways to conceive of traits that don't make them subclasses but I'm not sure of a non-awkward way to do this

**Comment by dosumis** [comment]
I had high hopes that COB could play a role that hides all the philosophy - with a bunch of classes that are biologist-facing. Is there still some prospect of getting classes that fulfill this role?  - in this case having a phenotype term which we could all share, hiding anything that sits above it for display purposes (preferred roots)?

**Comment by cmungall** [comment]
@dosumis - that's exactly the idea, but what is the implementation? One is to simply place phenotype under characteristic, but how do we resolve the longstanding dichotomy I highlighted? We could just continue to ignore it... but we at least need a consensus among phenotype ontologies that subclassing characteristic is in accord with their own way of thinking.

**Comment by cmungall** [comment]
or are you suggesting phenotype is a root and we don't make any commitment towards characteristic, ME, process...? (which is not incompatible with the Schulz-union model)

**Comment by dosumis** [comment]
I think we should make the commitment.  I think it would be nice if COB also gave us a biologist-facing term we can use to hide it.

**Comment by dosumis** [comment]
I do like the idea of being explicit about 'aggregates of characteristics' somewhere.  This is what we're planning (might already be doing) for some compound phenotypes.  I'm not sure where that puts us in BFO though.  Can characteristics have parts?

**Comment by mellybelly** [comment]
I disagree that traits are not phenotypes. In my opinion, these are synonyms but the terms are used differently by different communities. here are a few examples:

Trait- used in agriculture and biodiversity to describe more often quantitative measures, but not always and sometimes phenotype is used in agriculture as well, less often in biodiversity

phenotype- used frequently in model organism biology and usually in comparison to some standard or normal or wild type

Phenotype - used in clinical genetics to refer to a disorder/disease, they prefer 'phenotypic feature' as a synonym to the above traits/phenotypes

Biologically, a trait/phenotype (imho) is any observable process or physical manifestation (continuant) that is the result of the organism's genome + environment. Normal/abnormal is not relevant here imho- these are domain-specific and even context/experiment specific. 

Agree about hiding the philosophy - community will continue to (generally) go by something like the above. COB may as well embrace it rather than complicate it?

**Comment by dosumis** [comment]
Given this https://github.com/OBOFoundry/COB/issues/127 I'm happy to have:

characteristic 
  . phenotype

**Comment by wdduncan** [comment]
I find the discussion concerning phenotypes confusing. When I've talked to physicians sometimes they seem to be talking about a characteristic (i.e., quality, disposition) of a disease, sometimes the observation manifestation of a disease (e.g., facial tics are a phenotype of Tourette syndrome, or the presence of tumor is a phenotype of cancer), sometimes all of the above, and sometimes phenotype is not necessarily associated with disease.   
Given this, would it be better to have phenotype as a relation between the characteristics/processes/material entities and the things expressed?

**Comment by matentzn** [comment]
I guess you asking that kind of reveals that I just don't understand enough the purpose of COB. I only ever thought about two things:
1. there are phenotype classes (in HP, MP, and dozens of others)
2. they somehow relate to other things (PATO characteristics, bearers like anatomical entities, diseases etc)

Somehow I want to be able to say:
"this relation has as domain phenotype and as range disease."

I didn't really consider the deeper modelling implications here.. Do you see where I am coming from? Maybe phenotype is really too complicated to be covered by COB!

**Comment by wdduncan** [comment]
@matentzn 
Yes, I see where are coming from :)  
IMHO, the word 'phenotype' has so many uses that there may not be a "thing" that can serve as a domain/range. Rather, all the uses of 'phenotype' constitute a [family of resemblances](https://en.wikipedia.org/wiki/Family_resemblance). Or, as you suggest, it might be too complicated for COB.   
This is why I suggested using a relation (e.g., facial tics `phenotype of` Tourette syndrome). Taking this approach we create a logically defined class `phenotype`, if we find it is useful to have such a class.

**Comment by matentzn** [comment]
Thanks for the suggestion. I will contemplate it further, and will come back to you!

**Comment by johnwjudkins** [comment]
I agree with phenotype as 'characteristic'. Entities like an increased frequency of coughing, skin discoloration, or facial tics fit the Nature definition and are also characteristics of the organism (or a part thereof, or of a biological process of the organism). A tumorous quality of a tissue can be understood as a phenotype (even though the tumor itself is a material entity). By the way, OGMS' definition of phenotype agrees with the statement that a characteristic can have characteristics as parts.
